tests/seed.test.js

```diff
@@ -0,0 +1,11 @@
+const dataGen = require('../server/dataGen.js').dataGen;
```

Comment:
You may use destructuring to require both dataGen and Home at once: `{dataGen, Home}`

Reply:
Used destructuring.